common/source/docs/common-lua-scripts.rst

```diff
@@ -165,11 +171,17 @@ The following operators are supported on a Vector3f:
 
 - :code:`-` - Subtracts the two vectors by components.
 
-AHRS
-~~~~
+AHRS (ahrs:)
+~~~~~~~~~~~~
 
 The ahrs library represents the Attitude Heading Reference System computed by the autopilot. It provides estimates for the vehicles attitude, and position.
 
+- :code:`get_roll()` - Returns the current vehicle roll angle in degrees.
+
+- :code:`get_pitch()` - Returns the current vehicle pitch angle in degrees.
```

Comment:
Radians, you need to actually lookup the code when writing the descriptions.

Reply:
fixed